packages/protocol-deployments/test/ZoraCreator1155SupplyRoyaltyDeprecation.t.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "forge-std/Test.sol";
+
+import {
+    ForkDeploymentConfig,
+    Deployment,
+    ChainConfig
+} from "@zoralabs/zora-1155-contracts/src/deployment/DeploymentConfig.sol";
+import {ZoraDeployerUtils} from "@zoralabs/zora-1155-contracts/src/deployment/ZoraDeployerUtils.sol";
+
+import {Zora1155Factory} from "@zoralabs/zora-1155-contracts/src/proxies/Zora1155Factory.sol";
+import {ZoraCreator1155FactoryImpl} from "@zoralabs/zora-1155-contracts/src/factory/ZoraCreator1155FactoryImpl.sol";
+import {ZoraCreator1155Impl} from "@zoralabs/zora-1155-contracts/src/nft/ZoraCreator1155Impl.sol";
+import {ICreatorRoyaltiesControl} from "@zoralabs/zora-1155-contracts/src/interfaces/ICreatorRoyaltiesControl.sol";
+import {IMinter1155} from "@zoralabs/zora-1155-contracts/src/interfaces/IMinter1155.sol";
+
+contract ZoraCreator1155SupplyRoyaltyDeprecationTest is Test, ForkDeploymentConfig {
+    address internal creator;
+    string[] internal chains;
+
+    function setUp() public {
+        creator = makeAddr("creator");
+
+        chains = new string[](1);
+        chains[0] = "zora_goerli";
+
+        // chains = new string[](2);
```

Comment:
why is this commented out? can it just be deleted?

Reply:
oops fixed now